resources/git-fork.md

```diff
@@ -0,0 +1,98 @@
+# Git: Forking a repository
+
+> _Ah, the life! Branching, committing, pushing, pulling, merging... modifying code to my hearts' content... breathing life into features like some sort of magician or squashing bugs like an exterminator. Bliss. Until..._
+
+```bash
+! [remote rejected] master -> master (protected branch hook declined)
+error: failed to push some refs to 'git@github.com:some-account/some-repo.git'
+```
+
+☹️
+
+When working with code, there are times when you will want to modify code that resides in a repository that is not under your control, or with which for a variety of reasons you don't wish to interact. You might want to introduce features that are outside the scope of the project. You may want to build something completely unrelated and just use the repository as a starting point. Maybe a useful library needs a bug fix, but the maintainer has abandoned the repository. 
```

Comment:
` with which for a variety of reasons you don't wish to interact` is a bit hard to read, how about something like `or you may just not want to touch the source code directly` ?

Reply:
I should do less writing after midnight 😳 

Changed - thanks